### ISSUE TITLE
Possible error when computing final loss

### ISSUE THREAD

**Issue by secsilm** [dair-iitd/openie6#10]
In the paper, the final loss is obtained by adding constrained loss to cross entropy:

![image](https://user-images.githubusercontent.com/8894643/128845538-d515ec0f-dc45-4ea1-aa84-dd3ee1721c0c.png)

But in [`model.py`](https://github.com/dair-iitd/openie6/blob/master/model.py#L162), it seems that only the constrained loss is used:

![image](https://user-images.githubusercontent.com/8894643/128846685-ac69dd04-df68-4136-bbd1-1e6bc164ec50.png)

If I understand the paper correctly, should `loss = const_loss` be changed to `loss += const_loss`?

**Comment by SaiKeshav** [comment]
Hi, unfortunately, this does seem to be a bug in the constrained training portion of the code - thanks for pointing it out. It still seems to work because the constraints are applied after the warmup phase where the model is already trained with the Cross-Entropy loss and hence has good knowledge of it, already. Since the reported results can be replicated with this version of the code only, I am hesitant to change the code. I will update the README to reflect this issue. My apologies for the confusion!
